### DISCUSSION TITLE
belarusian language

### DISCUSSION THREAD

**Discussion by CarManBelarus** [home-assistant/frontend#18540] [Localization]
Please add support of belarusian language:
English Name: Belarusian
Native Name: Беларуская

**Comment by itbeard** [comment]
Please add our language :)

**Comment by iamwavecut** [comment]
\+

**Comment by CarManBelarus** [comment]
Thank you for enabling Belarusian for Android! Would be nice to have it in frontend and backend projects as well.

**Comment by alehmazol** [comment]
Please add support for Belarusian language in frontend and backend projects.
